lib/govspeak.rb

```diff
@@ -224,7 +224,7 @@ def insert_strong_inside_p(body, parser = Govspeak::Document)
       text = text.strip
       href = href.strip
 
-      %(\n<a role="button" class="#{button_classes}" href="#{href}" #{data_attribute}>#{text}</a>\n)
+      %(\n<a role="button" draggable="false" class="#{button_classes}" href="#{href}" #{data_attribute}>#{text}</a>\n)
```

Comment:
We still need `draggable="false"` here right?

Reply:
After a very quick test, we indeed do.